ext/DynamicQuantitiesLinearAlgebraExt.jl

```diff
@@ -1,8 +1,69 @@
 module DynamicQuantitiesLinearAlgebraExt
 
-import LinearAlgebra: norm
-import DynamicQuantities: AbstractQuantity, ustrip, dimension, new_quantity
+import LinearAlgebra: norm, inv, (\), svd, Algorithm, default_svd_alg, SVD, Diagonal, Adjoint, Transpose, AbstractRotation, AbstractMatrix, eigen, eigsortby, Eigen, det
+import DynamicQuantities: AbstractQuantity, ustrip, dimension, new_quantity, AbstractDimensions, QuantityArray, Quantity
+
+const QuantityArrayVecOrMat{T} = Union{QuantityArray{T,2},QuantityArray{T,1}} where T
 
 norm(q::AbstractQuantity, p::Real=2) = new_quantity(typeof(q), norm(ustrip(q), p), dimension(q))
 
+\(q::QuantityArrayVecOrMat,r::QuantityArrayVecOrMat) = QuantityArray(ustrip(q)\ustrip(r),dimension(r)/dimension(q))
+\(q::QuantityArrayVecOrMat,r::Union{AbstractVector,AbstractMatrix}) = QuantityArray(ustrip(q)\r,inv(dimension(q)))
+# not implemented, AbstractMatrix \ QuantityArray
+
+inv(Q::QuantityArray{T,2}) where T = QuantityArray(inv(ustrip(Q)),inv(dimension(Q)))
+
+"""
+    svd(A::QuantityArray; full::Bool = false, alg::Algorithm = default_svd_alg(A)) -> SVD
+
+    Singular value decomposition (SVD) of `QuantityArray`.
+    Exists for uniform matrices which includes all `QuantityArray`s (pp. 124, Hart, 1995).
+
+    Returns SVD factorization of parametric type: `SVD{T, Quantity{T, Dimensions{DynamicQuantities.FixedRational{Int32, 25200}}}, Matrix{T}, QuantityArray{T, 1, Dimensions{DynamicQuantities.FixedRational{Int32, 25200}}, Quantity{T, Dimensions{DynamicQuantities.FixedRational{Int32, 25200}}}, Vector{T}}}`.
+
+    Factorization `F` can be deconstructed: `U,σ,V = F`. 
+
+    Functions working: , `size`, `adjoint`.
+    Functions partially working: `inv`
+    Functions not working: `svdvals`, `ldiv!`.
+"""
```

Comment:
I don't think these are needed, since it will already show up for `?svd`? Likewise for `eigen`.

Reply:
Oh wait maybe it is needed; is the behavior different at all?

Reply:
Same core behavior but some of the side functionality is not implemented. I think it makes sense to remove this and the `eigen` docstring. It could be good to leave a comment regarding non-implemented functionality.